tests/modules/cooler/dump/main.nf

```diff
@@ -9,5 +9,5 @@ workflow test_cooler_dump {
     input = [ [ id:'test' ], // meta map
               file("https://raw.githubusercontent.com/open2c/cooler/master/tests/data/toy.asymm.16.cool", checkIfExists: true) ]
 
-    COOLER_DUMP ( input )
+    COOLER_DUMP ( input, [:] )
```

Comment:
How is this change related to the `zoomify` module?

Reply:
I updated cooler/dump also. This code is not related with `zoomify`. But because in the test I need to use cooler/dump to check the output, this should be also change.

Reply:
I suppose this is resolved then?